apps/test/unit/templates/studioHomepages/RecentCoursesTest.js

```diff
@@ -57,7 +57,6 @@ describe('RecentCourses', () => {
                 link={courses[0].link}
                 isRtl={false}
               />
-              <div>.</div>
```

Comment:
This is another case where I wonder if the test is a bit fragile and is testing (or really just matching) implementation details more than actually testing the outcome of the code...